lib/components/ReactBingmaps/ReactBingmaps.js

```diff
@@ -107,9 +107,12 @@ var ReactBingmaps = function (_Component) {
 		key: 'componentWillReceiveProps',
 		value: function componentWillReceiveProps(nextProps) {
 			var mapReference = nextProps.id ? '#' + nextProps.id : '.react-bingmaps';
-			if (this.props.center.join() !== nextProps.center.join()) {
+			if (this.props.center && this.props.center.join() !== nextProps.center.join()) {
```

Comment:
Couldn't join() result in a false positive for the comparison?  LIke if the coords were [2, 22] and [22, 2], they would be the same when joined right?